gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/util/string/EncodeDecodeUtils.java

```diff
@@ -0,0 +1,20 @@
+package com.ca.apim.gateway.cagatewayexport.util.string;
+
+public class EncodeDecodeUtils {
```

Comment:
Do we need to have two `EncodeDecodeUtils` classes?

Reply:
No, but I'm waiting for the work related to refactoring to happen, I don't want to change the project dependencies in this PR. If it goes in first I'll rebase and remove one

Reply:
removed the version in gateway export plugin